pyQuARC/code/custom_validator.py

```diff
@@ -132,16 +143,18 @@ def doi_missing_reason_explanation(explanation, missing_reason, doi):
     @staticmethod
     @if_arg
     def boolean_check(field_value):
+        # Checks if the value is a boolean, basically one of 'true' or 'false'
         return {
             "valid": field_value.lower() in ["true", "false"],
             "value": field_value
         }
 
     @staticmethod
-    def collection_progress_consistency_check(ends_at_present_flag, ending_date_time, collection_state):
+    @if_arg
+    def collection_progress_consistency_check(collection_state, ends_at_present_flag, ending_date_time):
         # Logic: https://github.com/NASA-IMPACT/pyQuARC/issues/61
         validity = True
-        if collection_state.upper() == "ACTIVE":
+        if collection_state.upper() in ["ACTIVE", "IN WORK"]:
```

Comment:
Constants?

Reply:
we don't really use it elsewhere, so do we really need to?

Reply:
No worries then